packages/frontpage/drizzle.config.ts

```diff
@@ -12,11 +12,10 @@ if (!URL || !AUTH_TOKEN) {
 }
 
 export default defineConfig({
+  dialect: "turso",
```

Comment:
Is this correct for a dialect?

Reply:
https://orm.drizzle.team/docs/drizzle-config-file#dialect

Reply:
Oh nice, I did a quick check to see if I could find it in the docs but missed it.